fmralign/template_alignment.py

```diff
@@ -193,18 +193,19 @@ def __init__(self, alignment_method="identity", n_pieces=1,
         ----------
         alignment_method: string
             Algorithm used to perform alignment between X_i and Y_i :
-            * either 'identity', 'scaled_orthogonal', 'ridge_cv', \
-            'permutation', 'diagonal'
-            * or an instance of one of alignment classes \
-            (imported from functional_alignment.alignment_methods)
+            * either 'identity', 'scaled_orthogonal', 'optimal_transport',
+            'ridge_cv', 'permutation', 'diagonal'
+            * or an instance of one of alignment classes (imported from
+            functional_alignment.alignment_methods)
         n_pieces: int, optional (default = 1)
             Number of regions in which the data is parcellated for alignment.
             If 1 the alignment is done on full scale data.
-            If > 1, the voxels are clustered and alignment is performed \
-            on each cluster applied to X and Y.
+            If > 1, the voxels are clustered and alignment is performed on each
```

Comment:
Should we update this docstring to mention the interaction with `clustering` if it's a 3D Niimg ?